lambda/voting/frontend/src/Components/Voting/Voting.js

```diff
@@ -90,7 +97,7 @@ export const Voting = ({ sdk }) => {
               onChange={(e) => setVoterId(e.target.value)}
             />
           </label>
-          <input type="submit" value="Submit" />
+          <input type="submit" value="Get Candidate" />
```

Comment:
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/100057389/200930842-7ee065c8-a685-4aae-9bb6-338098743d34.png">

This page looks confusing.
Separate candidate details and voter email a little bit

Reply:
![Screenshot from 2022-11-10 02-02-04](https://user-images.githubusercontent.com/102578238/201001387-24abf7f1-0333-4a27-ba53-1433800107f7.png)

Reply:
fixed